utilities/transactions/write_unprepared_transaction_test.cc

```diff
@@ -335,7 +335,7 @@ TEST_P(WriteUnpreparedTransactionTest, RecoveryTest) {
           for (int i = 0; i < num_batches; i++) {
             ASSERT_OK(txn->Put("k" + ToString(i), "value" + ToString(i)));
             if (txn_options.write_batch_flush_threshold == 1) {
-              ASSERT_EQ(wup_txn->GetUnpreparedSequenceNumbers().size(), i + 1);
```

Comment:
This change needs some comments. Otherwise when the test fails, it would be difficult to argue about the correct value expected from GetUnpreparedSequenceNumbers.